content/guide/setup.md

```diff
@@ -9,11 +9,10 @@ The most obvious difference between Quire and other publishing programs you may
 
 This is working closer to the code. And to work with code, you need three standard coding tools. Before you start the install process. Set these up first:
 
-1. Get a **text editor** (there are many, but we recommend [Atom](https://atom.io/))
-2. Set up a **[GitHub account](https://github.com/join)** and download **[GitHub Desktop](https://desktop.github.com/)** for version control
-3. Make sure you have a **command line shell**. Macs come with a good shell called Terminal (find it in your Applications/Utilities folder). For PCs we recommend installing [Git Bash](https://git-for-windows.github.io/).
+1. Make sure you have a **command line shell**. Macs come with a good shell called **Terminal** (find it in your Applications > Utilities folder). For PCs we recommend installing **[Git Bash](https://git-for-windows.github.io/)**. A good introduction to the command line shell can be found in this [lesson](https://programminghistorian.org/lessons/intro-to-bash) from The Programming Historian.
```

Comment:
Good idea to link to the tutorial at the Programming Historian, and we should continue linking to external resources like this throughout the docs. I'd just like to use different language so that the sentence makes sense even if the link isn't a link for any reason. (Like if we were to print the docs out as a paperback POD.) As it is, without the link there, the phrase "can be found in this lesson from The Programming Historian" reads as incomplete. Make sense?

Perhaps something like: "For more, see "Introduction to the Bash Command Line" from the Programming Historian"

Reply:
Agreed